htdocs/mri/jiv/get_file.php

```diff
@@ -60,7 +30,6 @@ function resolvePath($path)
 // inline. They'll still show up in the Apache logs.
 ini_set("display_errors", "Off");
 
-require_once __DIR__ . "/../../../vendor/autoload.php";
```

Comment:
@driusan It's a duplicate from line 58. I can add back though if you really want...